README.md

```diff
@@ -158,14 +157,14 @@ Please refer to [Shutting down a domain](site/shutdown-domain.md) for informatio
 
 ## Load balancing with the Traefik Ingress controller
 
-The initial Technology Preview release of the operator supports only the Traefik load balancer/Ingress controller.  Support for other load balancers is planned in the future.
-Please refer to [Load balancing with Traefik](site/traefik.md) for information about current capabilities.
+This release of the operator supports only the Traefik load balancer/Ingress controller.  Support for other load balancers is planned in the future.
```

Comment:
Yes Traefik will not be the only load balancer we support.  Probably the sub title of this section should change to "Load balancing"
Have sub sections for Traefik, Voyager (Lily He will provide documentation), Apache (Dongbo will provide documentation)

Reply:
Ok. I put in the subsections Traefik, Voyager, and Apache and left Voyager and Apache blank for others to fill in. The Traefik section has the existing one line: Please refer to Load balancing with Traefik for information about current capabilities.